tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/NameStringProxy.cs

```diff
@@ -13,6 +13,16 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
     [DuckCopy]
     internal struct NameStringProxy
     {
-        public object Value;
+        public string Value;
+    }
+
+    /// <summary>
+    /// nullable structs need an explicit proxy
+    /// </summary>
```

Comment:
This one still blows my mind, I'd love to see some tests for it 😅

Reply:
I mean there are tests on ducktype [here](https://github.com/DataDog/dd-trace-dotnet/blob/e14cb1137d6aa96f6b20babccfacafd9364e0364/tracer/test/Datadog.Trace.DuckTyping.Tests/DuckChainingNullableTests.cs) but you mean tests for this specific one ?

Reply:
I really mean as part of the integration test, cases where `NameString` is null, and/or where `NameString.Value` is `null`. I don't know if that's practical, just changing duck types always makes me a bit nervous 😄

Reply:
I added a test where the nullable is null (I don't know how to make the NameString have no value though)

Reply:
Hmmm, there's no associated span in the new snapshots though are there? 🤔

Reply:
hmm, I think there is one, see the comment by github-actions below
but it's not very explicit that OperationName is null, I just know it because I put a breakpoint.

Reply:
ah, right. I'll double check what is happening

Reply:
Ok cool, so we _do_ already have tests for this, cool, thanks! 🙂